hs/src/Reach/Eval/Core.hs

```diff
@@ -1763,7 +1763,7 @@ evalForm f args = do
           go $ p {slpr_mpay = Just x}
         Just PRM_Def -> do
           x <- one_arg
-          go $ p {slpr_mdef = Just x}
+          go $ p {slpr_defs = x : slpr_defs}
```

Comment:
This puts it at the front, but I think it should put it at the end, so that they occur in the output in the order of the program. You should have an example where the second refers to something bound in the first and you'll see that it won't work